layouts/article/list-images.html

```diff
@@ -0,0 +1,65 @@
+{{ define "main" }}
+<div class="row flex-md-nowrap">
+    <div class="leftnav-container">
+        <div class="board-background leftnav">
+            <nav id="sidebar-default" aria-label="Main navigation">
+                {{ partial "sidebar/docs-menu.html" . }}
+            </nav>
+            {{ partial "sidebar/nav-bottom.html" }}
+        </div>
+    </div>
+    <div class="row justify-content-center topic-container flex-shrink-1">
+        <div class="col-md-12 col-lg-10 col-xl-8">
+            <article>
+                <h1 class="text-center mb-5">{{ if eq .CurrentSection .FirstSection }}{{ .Section | humanize }}{{ else }}{{ .Title }}{{ end }}</h1>
+                    {{ if eq .Params.type "article" }}
+                        <div class="container">
+                        <div class="row">
+                        {{ $currentSection := .CurrentSection }}
+                            {{ $i := 0 }}
+                            {{ $itemsPerRow := 4 }}
+                        {{ range and
+                            (where .Site.RegularPages.ByTitle "Section" .Section)
+                            (where .Pages ".Params.unlisted" "!=" "true")}}
+                            {{ if in (.RelPermalink | string) $currentSection.RelPermalink }}
+                                    <div class="col mb-4">
+                                        <div class="card">
+                                            <div class="card-body">
+                                                <h5 class="card-title"><a href="{{ .RelPermalink }}">{{ .Params.linktitle }}</a></h5>
+                                                <p class="card-text">{{ .Params.description }}</p>
+                                                <a href="{{ .RelPermalink }}image_specs/" class="card-link text-primary">Variants</a>

```

Comment:
Sidenote contrast checker complains about link colors, if you remove `.text-primary` from the links and edit _custom.scss to add a `.card-link` override you'll get the contrast-passing color;

```
// Beat <a> reset
.docs-content.docs-content a:not(.tag),
.card-link.card-link.card-link.card-link {
  color: var(--article-link-color);
}
```